synapse/http/federation/srv_resolver.py

```diff
@@ -28,7 +28,7 @@
 
 logger = logging.getLogger(__name__)
 
-SERVER_CACHE = {}
+SERVER_CACHE: Dict = {}
```

Comment:
Can we make this more specific? (What is this a mapping of?)

Reply:
An example
https://github.com/matrix-org/synapse/blob/4b965c862dc66c0da5d3240add70e9b5f0aa720b/tests/http/federation/test_srv_resolver.py#L77-L85